### PR TITLE
gui spacing fix for previous pull

### PR DESCRIPTION
**Pull request by zDEFz**

### CODE REVIEW

**Comment by kroyee** [comment]
This does not look right to me.
What is in the repo is already ok.

**Comment by zDEFz** [comment]
@kroyee . If you leave it like this then Rotate Left and Rotate Right will be displayed wrong -> **it would be cut off.** 
Why discard it?
btw it's already tested and it works proper.

File: gui.cpp [hunk 1]
```diff
@@ -785,7 +785,7 @@ void UI::minimize(tgui::ChildWindow::Ptr win) {
 
 void UI::changeServerAdd(sf::String addr) { //Quickfix for issue #33
 	std::string serveraddr = addr.toAnsiString();
-	if (serveraddr.back() != '.' || serveraddr.size() != 0)
+	if (!(serveraddr.back() == '.' || serveraddr.size() == 0))
```
Comment (rmcat) [inline gui.cpp:788]: This last change should be in a separate commit. #53 currently discusses it.

**Comment by kroyee** [comment]
Ok, I see it now.
Hehe, very confusing all of this.

File: gui.cpp (hunk 1 above)
Comment (kroyee) [reply gui.cpp:788]: I think this is here because the pull-request was started before I made that change in the repo. Hence it shows up as a diff in the commits.
I'm not sure though... but it's actually changing it to what is already in the repo (if that makes any sense)

Comment (kroyee) [reply gui.cpp:788]: What would make things more clear is if there was a "Merging with origin/master" commit to show the workflow.
